templates/account/payment-methods/index.html

```diff
@@ -28,53 +26,82 @@
+      <div id="default-payment-method-section"
+           class="row {% if not default_payment_method.id %}u-hide{% endif %}"
+           style="margin-top: 2rem">
         <div class="col-4 current-payment-method">
           {% with brand=default_payment_method["brand"] %}
             {% include "account/payment-methods/_card-logo.html" %}
           {% endwith %}
-          <p><span style="text-transform: capitalize;">{{default_payment_method["brand"]}}</span> ending in {{default_payment_method["last4"]}}</p>
-          </div>
+          <p>
+            <span style="text-transform: capitalize;">{{ default_payment_method["brand"] }}</span> ending in {{ default_payment_method["last4"] }}
+          </p>
+        </div>
         <p class="col-4 u-text--muted">Default payment method</p>
         <div class="col-4 u-align--right">
           <button class="p-button" id="edit-payment-details">Edit</button>
+          <button class="p-button--base"
+                  id="remove-payment-details"
+                  aria-controls="remove-payment-modal">Remove</button>
         </div>
       </div>
 
-      {% if default_payment_method %}
-        <div id="edit-payment-method-section" class="row {% if default_payment_method["id"] %}u-hide{% endif %}" style="margin-top: 2rem;">
-          <div class="col-8">
-              <div id="card-element"></div>
-              <span id="card-errors" class="p-form-validation__message u-hide"></span>
-          </div>
-          <div class="col-4 u-align--right">
-            {% if default_payment_method["id"] %}<button class="p-button" id="cancel-payment-details">Cancel</button>{% endif %}
-            <button class="p-button--positive" id="update-payment-details" disabled>Update</button>
+      <div id="edit-payment-method-section"
+           class="row u-hide"
+           style="margin-top: 2rem">
+        <div class="col-8">
+          <div id="card-element"></div>
+          <span id="card-errors" class="p-form-validation__message u-hide"></span>
+        </div>
+        <div class="col-4 u-align--right">
+          <button class="p-button" id="cancel-payment-details">Cancel</button>
+          <button class="p-button--positive" id="update-payment-details" disabled>Update</button>
+        </div>
+        <div class="col-8"></div>
+      </div>
+      <hr class="{% if not default_payment_method.id %}u-hide{% endif %}" />
+
+      <div id="no-payment-method-section"
+           class="p-strip {% if default_payment_method.id %}u-hide{% endif %}">
+        <div class="row">
+          <div class="u-align--right col-4">
+            <i style="width: 4rem; height: 4rem; margin-top: 0.2rem;" 
+               class="p-icon--credit-card u-hide--medium u-hide--small"></i>
           </div>
-          <div class="col-8">
+          <div class="u-align--left col-8 col-medium-4 col-small-3">
+            <p class="p-heading--4 u-no-margin--bottom">You have no payment method saved</p>
```

Comment:
I think `margin-bottom` should be 0. `margin-bottom` is  -0.45rem !important; because of this class `u-no-margin--bottom`

Reply:
I always though it was the same... Done, thanks!